packages/config-eslint/src/resolvers/graphql.ts

```diff
@@ -0,0 +1,11 @@
+import { GQL_EXT_PATTERN } from '@airbnb/nimbus-common';
+
+export const interfaceVersion = 2;
+
+export function resolve(source: string): { found: boolean; path?: unknown } {
```

Comment:
why `path?: unknown` vs `null`?

Reply:
Just because I don't know what eslint expects it to be.